src/util/validate.js

```diff
@@ -0,0 +1,11 @@
+/**
+ * Validate options
+ *
+ * @param  {object}  options      Options
+ * @param  {object}  releaseInfo  Version specific NW release info
+ * @return {boolean}              True if options are valid. False otherwise
+ */
+export const validate = async (options, releaseInfo) => {
+  if (options && releaseInfo) return true;
+  else return false;
+};
```

Comment:
This also shouldn't be `async`. Lots of these that need to be fixed.

Reply:
✅